README.md

```diff
@@ -1,2 +1,121 @@
-# defenz-alerts
-This script is used to send scheduled Defenz alerts by using Defenz APIs
+# Defenz Timely Alerts
+This script is used to generate and send block reports for a specific period of time. 
+This script can be scheduled by a scheduler e.g. cron to be run at specific times.
+
+`NOTE`: To run this script you need Python 3.6+
+
+## How to install
+Clone this repository:
+```shell script
+git clone https://github.com/InternetNZ/defenz-alerts.git
+```
+Install the dependencies:
+```shell script
+cd defenz-alerts
+pip3 install -r requirements.txt
+```
+
+## Set the configs
+Before running the script for the first time you need to configure the script. The config file
+is located in ``DEFENZ_ALERTS_DIR/defenz_alerts/config.ini``. If the config file does not exist 
+creat one from the sample one:
+
+```shell script
+cd DEFENZ_ALERTS_DIR/defenz_alerts/
+cp config.sample.ini config.ini
+```
+
+The config file should look like below:
+
+```ini
+[DEFENZ]
+USERNAME =
+PASSWORD =
+CLIENT_ID =
+CLIENT_SECRET =
+REPORT_EMAIL =
+LOGIN_ENDPOINT = https://dnsfirewall-auth.defe.nz/auth/realms/D-ZoneFireWall/protocol/openid-connect/token
+API_URL = https://dnsfirewall-api.defe.nz
```

Comment:
Would be useful an `INTERVAL` configuration with a fallback to 5 minutes (what you already have) if not provided

Reply:
The default value is already set in arguments. So if it's not provided it get the default value.

Reply:
I meant that would be useful to be able to set a different interval using the configuration file instead of only by command-line. I feel that interval would a very interesting configuration to tweak and set it differently than the default depending on the networks load.

Reply:
I can add it.